cmd/terramate/cli/cli.go

```diff
@@ -197,53 +197,46 @@ func newCLI(
 		return nil, fmt.Errorf("failed to parse cli args %v: %v", args, err)
 	}
 
-	logLevel := parsedArgs.LogLevel
-	logFmt := parsedArgs.LogFmt
+	configureLogging(parsedArgs.LogLevel, parsedArgs.LogFmt, stderr)
 
-	configureLogging(logLevel, logFmt, stderr)
+	logger := log.With().Logger()
 
-	log.Trace().
-		Str("action", "newCli()").
-		Msg("Get working directory.")
-	wd := parsedArgs.Chdir
-	if wd == "" {
-		wd, err = os.Getwd()
-		if err != nil {
-			return nil, fmt.Errorf("failed to get working directory: %w", err)
-		}
-	}
-
-	logger := log.With().
-		Str("action", "newCli()").
-		Str("stack", wd).
-		Logger()
-
-	logger.Trace().
-		Msgf("Evaluate symbolic links for %q.", wd)
-	wd, err = filepath.EvalSymlinks(wd)
+	cwd, err := os.Getwd()
 	if err != nil {
-		return nil, fmt.Errorf("failed evaluating symlinks for %q: %w", wd, err)
+		log.Fatal().
+			Err(err).
+			Msg("Getwd() failed")
 	}
 
-	logger.Trace().
-		Msgf("Get absolute file path of %q.", wd)
-	wd, err = filepath.Abs(wd)
-	if err != nil {
-		return nil, fmt.Errorf("getting absolute path of %q: %w", wd, err)
-	}
+	if parsedArgs.Chdir != "" {
+		logger.Debug().
+			Str("cwd", cwd).
+			Str("dir", parsedArgs.Chdir).
+			Msg("Changing working directory")
+		err = os.Chdir(parsedArgs.Chdir)
+		if err != nil {
+			logger.Fatal().
+				Str("cwd", cwd).
+				Str("dir", parsedArgs.Chdir).
+				Err(err).
+				Msg("Changing working directory failed")
+		}
 
-	logger.Trace().
-		Msgf("Change working directory to %q.", wd)
-	err = os.Chdir(wd)
-	if err != nil {
-		return nil, fmt.Errorf("failed to change working directory to %q: %w", wd, err)
+		cwd, err = os.Getwd()
```

Comment:
```suggestion
		cwd = parsedArgs.Chdir
```

Should be safe since we just called "os.Chdir" with success (no need to call Getwd again, I think).

Reply:
getwd returns the absoulte path.... chdir can be relative and is invalid after the chdir was executed